simple_history/tests/tests/test_models.py

```diff
@@ -627,6 +627,16 @@ def test_create_history_model_with_multiple_one_to_ones(self):
                       "fields to one to one fields without throwing an "
                       "exception.")
 
+    def test_create_history_model_with_custom_model_name(self):
+        records = HistoricalRecords(custom_model_name='MyHistoricalBookcase')
```

Comment:
Maybe instantiate a model with a custom_model_name in one of the test models files?